api/docs/scatter_gather_emulation.dox

```diff
@@ -71,6 +74,126 @@ register `k1`. Elements may be scattered in any order. When an element is stored
 its mask is cleared. If some store faults, all elements to its right (closer to
 LSB) will be complete.
 
+## AArch64
+
+The AArch64 SVE (Scalable Vector Extension) introduced the first AArch64 scatter and
```

Comment:
A link to the manual would be nice here.

Reply:
Done.